service/controller/clusterapi/v29/resource/tccp/testdata/case-0-basic-test.golden

```diff
@@ -1085,7 +993,4 @@ Resources:
         MaxBatchSize: 1
         # after creating a new instance, pause operations on the ASG for this amount of time
         PauseTime: PT15M
-
-  
-
-
+  
```

Comment:
Same problem here. Uncertain why that is. There should only be new lines be removed from the rendered template. So we see the integrity of the rendering is ensured.